indexer/locality_index_builder.hpp

```diff
@@ -0,0 +1,59 @@
+#pragma once
+#include "indexer/cell_id.hpp"
+#include "indexer/cell_value_pair.hpp"
+#include "indexer/feature_covering.hpp"
+#include "indexer/interval_index_builder.hpp"
+#include "indexer/locality_object.hpp"
+#include "indexer/scales.hpp"
+
+#include "coding/dd_vector.hpp"
+#include "coding/file_sort.hpp"
+#include "coding/writer.hpp"
+
+#include "base/logging.hpp"
+#include "base/macros.hpp"
+#include "base/scope_guard.hpp"
+
+#include "defines.hpp"
+
+#include <string>
+#include <vector>
+
+namespace covering
+{
+template <class TObjectsVector, class TWriter>
+void BuildLocalityIndex(TObjectsVector const & objects, TWriter & writer,
```

Comment:
I feel that we need a description somewhere what LocalityIndex and LocalityObject are.
For instance, there is a LocalityIndex in search/ that means a completely different thing (and that we probably should now rename).

Reply:
I'm going to add LocalityIndex class with description. Can You suggest another name?
OsmIndex ? LocalityGeoIndex?

Reply:
LocalityIndex is ok.

Reply:
Will this class be added in another PR?

Reply:
yes